var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -94,6 +94,9 @@ class Abinit(AutotoolsPackage):
     # conflicts('+elpa', when='~mpi')
     # conflicts('+elpa', when='+scalapack')
 
+    patch('rm_march_settings.patch', when='%fj')
```

Comment:
Can we filter these settings for every platform? Spack compiler wrappers should be able to inject microarchitecture specific flags automatically.

Reply:
OK. I fixed it!